capstone-frontend/src/App.test.js

```diff
@@ -2,8 +2,15 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+var container = document.createElement("div");
+document.body.appendChild(container);
+
+test('renders sidebar', () => {
+ render(<App />, container);
+  const leftSidebar = document.getElementById("left-sidebar-container");
+  expect(leftSidebar).toBeInTheDocument();
+  const rightSidebar = document.getElementById("right-sidebar-container");
+  expect(rightSidebar).toBeInTheDocument();
+  const navbar = document.getElementById("navbar");
+  expect(navbar).toBeInTheDocument();
```

Comment:
Nice to see the React tests are working now. 👍

Reply:
Agreed!